doc/tutorial/didi/carprice_xgboost/README.md

```diff
@@ -1,32 +1,35 @@
-# XGBoost on SQLFlow Tutorial
+# Predict the Car Price using XGBoost in SQLFlow
+
+This tutorial describes how to train an XGBoost model using the [Cars Dataset](https://www.kaggle.com/CooperUnion/cardataset), then how to predict the car price using SQLFlow. You can checkout the [User Guide](https://github.com/sql-machine-learning/sqlflow/blob/develop/doc/user_guide.md) if you are new to SQLFlow. In this tutorial, you will learn how to:
```

Comment:
**check out** vs **checkout**

> Check out is a verb phrase that means to sign for something or to observe something. Checkout means a place to buy things or the process of leaving a hotel as a noun, and it describes qualities of these things as an adjective.

So in our documentation, we should use check out. :)